### PR TITLE
Introspection: remove alphabetical ordering of columns

### PR DESCRIPTION
**Pull request by carmenberndt**
related to https://github.com/prisma/prisma/issues/2554
closes https://github.com/prisma/prisma/issues/2951
Removes alphabetical ordering for MySql, PostgreSql and SQLite.
Removes alphabetical ordering in migration-engine.
Adjusts tests to new ordering.

### CODE REVIEW

File: libs/sql-schema-describer/src/mysql.rs [hunk 1]
```diff
@@ -189,8 +189,7 @@ async fn get_all_columns(
                 table_name table_name
             FROM information_schema.columns
             WHERE table_schema = ?
-            ORDER BY column_name
-            COLLATE utf8_general_ci
+            ORDER BY ordinal_position
```
Comment (mavilein) [inline libs/sql-schema-describer/src/mysql.rs:192]: Why was the `COLLATE` there before? Why don't we need it anymore?

Comment (mavilein) [reply libs/sql-schema-describer/src/mysql.rs:192]: I guess the answer to the second question is that we are ordering by a numeric column now :-)

File: libs/sql-schema-describer/src/sqlite.rs [hunk 2]
```diff
@@ -215,7 +215,6 @@ impl SqlSchemaDescriber {
                 col
             })
             .collect();
-        cols.sort_unstable_by_key(|col| col.name.clone());
```
Comment (mavilein) [inline libs/sql-schema-describer/src/sqlite.rs:218]: 🔥

File: migration-engine/connectors/sql-migration-connector/src/sql_schema_calculator.rs [hunk 3]
```diff
@@ -39,13 +39,6 @@ impl<'a> SqlSchemaCalculator<'a> {
 
         tables.append(&mut relation_tables);
 
-        // guarantee same sorting as in the sql-schema-describer
-        for table in &mut tables {
-            table
-                .columns
-                .sort_unstable_by(|a, b| a.name.as_str().cmp(b.name.as_str()));
```
Comment (mavilein) [inline migration-engine/connectors/sql-migration-connector/src/sql_schema_calculator.rs:46]: 🔥

File: libs/sql-schema-describer/src/mysql.rs (hunk 1 above)
Comment (carmenberndt) [reply libs/sql-schema-describer/src/mysql.rs:192]: The sorting rules were necessary because before string values were compared (`ORDER BY column_name`).

**Pull request merged**